content-repo/extra-docs/articles/splunk-add-on.md

```diff
@@ -0,0 +1,153 @@
+---
+id: splunk-add-on
+title: Demisto Add-on for Splunk
+---
+Supporting Add-on for Cortex XSOAR. This application allows to push incidents into Cortex XSOAR from Splunk according to configurable trigger parameters.
+
+
+### Prerequisites
+* Splunk version 7.2 >=
+
+* In order to use a Splunk Cloud instance, you need to contact Splunk support and ask them to open API access and use the non SAML authentication with it, as stated in this message from Splunk:
+  > Use the following URL for Splunk Cloud deployments. If necessary, submit a support case to open port 8089 on your deployment. Please include the IP Addresses/CIDR Ranges you would like to have access from.<br/>
+    https://\<deployment-name\>.splunkcloud.com:8089
+  
+  For additional information see: https://docs.splunk.com/Documentation/SplunkCloud/latest/RESTTUT/RESTandCloud
+
+
+### Prepare a local Splunk Environment
+Run the following command to create a Splunk docker container (replace the `*****` with any 8-character password, containing letters and digits):
+```
+docker run -d -p "8000:8000" -p "8088:8088" -p "8089:8089" -e "SPLUNK_START_ARGS=--accept-license" -e "SPLUNK_PASSWORD=*****" --name splunk splunk/splunk:latest
+```
+Once executed, the splunk env will be available at http://localhost:8000.
+
+
+### Installation of the Add-on
+* Download Demisto Add-on for Splunk from [Splunkbase](https://splunkbase.splunk.com/app/3448).
+* After initializing the container, open your local Splunk environment.
+* Go to "Manage Apps" → Install app from file → upload the latest version of Demisto Add-on for Splunk.
+  *Note:* if a version of the app already exists, mark the "Upgrade app" checkbox.
+  
+  ![install](https://user-images.githubusercontent.com/38749041/103541256-db406b80-4ea3-11eb-9280-279f50e447f6.gif)
+* Restart Splunk and login again.
```

Comment:
@dantavori  Is this really required?

Reply:
yes, you are redirected to this page after submitting the add-on file: 
![image](https://user-images.githubusercontent.com/38749041/109472086-5a07ee00-7a7a-11eb-88ec-713f216673d8.png)